pkg/services/ngalert/store/transactions.go

```diff
@@ -2,11 +2,6 @@ package store
 
 import "context"
 
-// TransactionManager represents the ability to issue and close transactions through contexts.
-type TransactionManager interface {
```

Comment:
Push this up a layer - more compliant with Go's style.

Avoided this originally because the layer above did not exist until this PR.